stdlib/bytes.ml

```diff
@@ -15,20 +15,20 @@
 
 (* Byte sequence operations *)
 
-external length : bytes -> int = "%string_length"
+external length : bytes -> int = "%bytes_length"
 external string_length : string -> int = "%string_length"
-external get : bytes -> int -> char = "%string_safe_get"
-external set : bytes -> int -> char -> unit = "%string_safe_set"
+external get : bytes -> int -> char = "%bytes_safe_get"
+external set : bytes -> int -> char -> unit = "%bytes_safe_set"
 external create : int -> bytes = "caml_create_string"
```

Comment:
see my comments above:

```ocaml
external caml_create_string : int -> bytes
external caml_fill_string : bytes -> int -> int -> char -> unit
```

indeed, the name is a bit misleading, either we stick to the name for better backward compatibility or we change the c primitive names or we add a duplication, I am happy with either, @hhugo @damiendoligez what do you think